csharp/src/Ice/IObjectPrx.cs

```diff
@@ -97,14 +97,17 @@ public static class Request
         /// pseudo-interface Object.</summary>
         public static class Response
         {
-            /// <summary>The <see cref="InputStreamReader{T}"/> reader for operation ice_id.</summary>
+            /// <summary>The <see cref="InputStreamReader{T}"/> reader for the return type of operation ice_id.
+            /// </summary>
             public static readonly InputStreamReader<string> IceId = InputStream.IceReaderIntoString;
 
-            /// <summary>The <see cref="InputStreamReader{T}"/> reader for operation ice_ids.</summary>
+            /// <summary>The <see cref="InputStreamReader{T}"/> reader for the return type of operation ice_ids.
+            /// </summary>
             public static readonly InputStreamReader<string[]> IceIds =
                 istr => istr.ReadArray(minElementSize: 1, InputStream.IceReaderIntoString);
 
-            /// <summary>The <see cref="InputStreamReader{T}"/> reader for operation ice_isA.</summary>
+            /// <summary>The <see cref="InputStreamReader{T}"/> reader for the return type of operation ice_isA.
+            /// </summary>
```

Comment:
Did you mean to change these lines in IObjectPrx? And if so, why did you only fix the formatting for Response? There's other lines in the file that have the same format but were unaltered. Smells like a errorneous merging issue to me...

Reply:
I did change the comment, not just the formatting.